beta/iam/guides/how-to-messaging-delegate.md

```diff
@@ -326,7 +326,30 @@ func shouldShowMessage(message: Showable) -> Bool {
 
 {% tab title="Android" %}
 
-Feature not yet available.
+If the developer would like to manually integrate the `View` that contains the UI for an in-app message, they can do so by accessing the `WebView` directly in a `MessagingDelegate` method.  
```

Comment:
Not a huge fan of the third-person usage here.

I'd go with: "If you would like to manually integrate the `View` that contains the UI for an in-app message, you can do so by accessing..."

Reply:
i'll fix this up throughout the messaging delegate doc page as well